submarine-server/server-submitter/submitter-k8s/src/main/java/org/apache/submarine/server/submitter/k8s/K8sSubmitter.java

```diff
@@ -683,6 +703,30 @@ private IngressRouteSpec parseIngressRouteSpec(String namespace, String name) {
     return spec;
   }
 
+  private void rollbackCreationNotebook(String pvName) {
```

Comment:
I think the function name is a little confusing. Maybe we will have other services that need rollback creation in the future. So I think it's better to change to below. 
```suggestion
  private void deletePersistentVolume(String pvName) {
```